storage/watchable_store_test.go

```diff
@@ -180,8 +185,15 @@ func TestSyncWatchings(t *testing.T) {
 	// All of the watchings actually share one channel
 	// so we only need to check one shared channel
 	// (See watcher.go for more detail).
-	if len(w.(*watcher).ch) != watcherN {
-		t.Errorf("watched event size = %d, want %d", len(w.(*watcher).ch), watcherN)
+	evs := <-w.(*watcher).ch
```

Comment:
why do we remove the check of len(w.(*watch).ch)? I expect this still to be watcherN. No?